src/schemaType/__tests__/ArrayType.test.ts

```diff
@@ -224,6 +227,6 @@ describe('validate', () => {
 		const value = [1.23, 4.56, 7.89];
 
 		const validationResult = await arrayType.validate(value, documentMock);
-		expect(validationResult).toHaveLength(0);
+		expect(validationResult).toEqual(new Map<string, string[]>());
```

Comment:
This can probably just be `expect(validationResult.size).toBe(0)` and accomplish the same goal without constructing an empty map.